src/openms/include/OpenMS/KERNEL/SpectrumHelper.h

```diff
@@ -33,6 +33,9 @@
 // --------------------------------------------------------------------------
 
 #include <OpenMS/MATH/STATISTICS/StatisticFunctions.h>
+#include <OpenMS/METADATA/DataArrays.h>
+#include <OpenMS/KERNEL/MSSpectrum.h>
+#include <OpenMS/KERNEL/MSChromatogram.h>
```

Comment:
MSChromatogram is not required, right?

and `#pragma once` should come first :)